hack/copy-kube-artifacts.sh

```diff
@@ -24,20 +26,26 @@ rsync -av \
   --include-from=- \
   --exclude='*' \
   $KUBE_ROOT/ $KUBE_GODEP_ROOT <<EOF
+test/e2e/**/*.yaml
+test/e2e/**/*.json
 api/swagger-spec/v1.json
 cmd/integration/**.json
 cmd/integration/**.yaml
 docs/admin/**.json
 docs/admin/**.yaml
-docs/user-guide/**.json
-docs/user-guide/**.yaml
+docs/user-guide/**/*.json
```

Comment:
is this going to skip the json/yaml in the root of the user-guide directory?  Looks like there are quite a few but not sure if we need them or not